undefined reference to `GB_JITpackage_nfiles`

**Describe the bug**
graphblas fails to link when cross-compiling, it seems some native target generating that source file is not triggered:
https://github.com/DrTimothyAldenDavis/SuiteSparse/blob/dev/GraphBLAS/JITpackage/CMakeLists.txt#L68


**To Reproduce**
use mingw from linux

**Expected behavior**
links succeeds

**Screenshots**
```
[ 95%] Linking C shared library libgraphblas.dll
/usr/lib/gcc/i686-w64-mingw32/13.2.1/../../../../i686-w64-mingw32/bin/ld: CMakeFiles/GraphBLAS.dir/objects.a(GB_jitifyer.c.obj):GB_jitifyer.c:(.text+0x218e): undefined reference to `GB_JITpackage_nfiles'
/usr/lib/gcc/i686-w64-mingw32/13.2.1/../../../../i686-w64-mingw32/bin/ld: CMakeFiles/GraphBLAS.dir/objects.a(GB_jitifyer.c.obj):GB_jitifyer.c:(.text+0x21a7): undefined reference to `GB_JITpackage_index'
/usr/lib/gcc/i686-w64-mingw32/13.2.1/../../../../i686-w64-mingw32/bin/ld: CMakeFiles/GraphBLAS.dir/objects.a(GB_jitifyer.c.obj):GB_jitifyer.c:(.text+0x21d4): undefined reference to `GB_JITpackage_nfiles'
/usr/lib/gcc/i686-w64-mingw32/13.2.1/../../../../i686-w64-mingw32/bin/ld: CMakeFiles/GraphBLAS.dir/objects.a(GB_jitifyer.c.obj):GB_jitifyer.c:(.text+0x21e5): undefined reference to `GB_JITpackage_index'
/usr/lib/gcc/i686-w64-mingw32/13.2.1/../../../../i686-w64-mingw32/bin/ld: CMakeFiles/GraphBLAS.dir/objects.a(GB_jitifyer.c.obj):GB_jitifyer.c:(.text+0x2279): undefined reference to `GB_JITpackage_nfiles'
collect2: error: ld returned 1 exit status
```

**Desktop (please complete the following information):**
 - OS: Linux
 - compiler mingw-w64-gcc 13.2.1
 - netlib blas/lapack 3.12.0
 - Version 7.5.0

**Additional context**
configuring with cmake -DSUITESPARSE_DEMOS=OFF -DBUILD_TESTING=OFF -DBLA_VENDOR=Generic ..., make install

succeeds if I disable jit: cmake -DGRAPHBLAS_USE_JIT=OFF

/cc @mmuetzel

--- a/GraphBLAS/JITpackage/CMakeLists.txt
+++ b/GraphBLAS/JITpackage/CMakeLists.txt
@@ -29,6 +29,11 @@ if ( CMAKE_CROSSCOMPILING )
         CMAKE_ARGS ${GRAPHBLAS_CROSS_TOOLCHAIN_FLAGS_NATIVE}
         INSTALL_COMMAND "" )
 
+    # Overriding this variable seems to be necessary if users set
+    # CMAKE_FIND_ROOT_PATH_MODE_PACKAGE to ONLY.
+    # See: https://github.com/DrTimothyAldenDavis/SuiteSparse/issues/729
+    set( GrBJITPackageGenerator_DIR ${PROJECT_BINARY_DIR}/native )
+
     # The following package won't be found on the first run (before the native
     # grb_jitpackage has been built). But it should be found after the above
     # external project has built it.

